Makefile

```
@@ -1,8 +1,7 @@
 VERSION?=3.1
 
 clean:
-	rm -rf build
-	rm -rf DenyHosts/*.pyc
+	git clean -fdx
```

Can we add this back in for the instances where this has not been git cloned, but downloaded via a zip file.
```suggestion
	git clean -fdx
	rm -rf build
	rm -rf DenyHosts/*.pyc
```